app/src/main/java/org/mozilla/fenix/home/sessioncontrol/viewholders/pocket/PocketStoriesComposables.kt

```diff
@@ -73,20 +87,53 @@ fun PocketStory(
         Card(
             elevation = 6.dp,
             shape = RoundedCornerShape(4.dp),
-            modifier = Modifier
-                .size(160.dp, 87.dp)
-                .padding(bottom = 8.dp)
+            modifier = Modifier.size(160.dp, 87.dp)
         ) {
-            // Don't yet have a easy way to load URLs in Images.
-            // Default to a solid color to make it easy to appreciate dimensions
-            Box(Modifier.background(Color.Blue))
-            // Image(
-            //     painterResource(R.drawable.ic_pdd),
-            //     contentDescription = "hero image",
-            //     contentScale = ContentScale.FillHeight,
-            // )
+            ImageLoader(
+                client = client,
+                // The endpoint allows us to ask for the optimal resolution image.
+                url = story.imageUrl.replace(
+                    "{wh}",
+                    with(LocalDensity.current) {
+                        "${160.dp.toPx().roundToInt()}x${87.dp.toPx().roundToInt()}"
+                    }
+                ),
+                targetSize = 160.dp
+            ) {
+                WithImage { painter ->
+                    Image(
+                        painter,
+                        modifier = Modifier.size(160.dp, 87.dp),
+                        contentDescription = "${story.title} story image"
+                    )
+                }
+
+                Placeholder {
+                    Box(
+                        Modifier.background(
+                            when (isSystemInDarkTheme()) {
+                                true -> Color(0xFF42414D) // DarkGrey30
+                                false -> PhotonColors.LightGrey30
+                            }
```

Comment:
nit: We use this twice here, do you expect this to be needed more often? Maybe we want to move this to some extension method or composable function or will this be a property on the theme? 🤔

Reply:
I think this placeholder / fallback image approach will indeed be the default for future Images so it definitely needs to be standardized.
Maybe we can have a reusable `..Image` composable to which users would pass just the ImageLoader needed arguments and the placeholder / fallbacks are hardcoded.